### ISSUE TITLE
Financial situation of The Matrix.org Foundation

### ISSUE THREAD

**Issue by alohapersona** [matrix-org/matrix-spec#571]

**Comment by alohapersona** [comment]
Hello there. It's me again.

One year has passed and I'm still not aware of any publication of financial situation reports of The Matrix.org Foundation beyond what is required to be filed by law. 

At the same time, The Matrix.org Foundation is asking the public for donations and membership fees, with membership tiers of up to 500k£. Without providing financial reports, it's impossible for the wider community to evaluate the needs of The Matrix.org Foundation and therefor attest if such requests for additional funding are justified.

Given that most of The Matrix.org Foundation's activities according to their rules outlined in MSC1779 - except for sub-contracting the hosting of matrix.org home server - are mostly organisational, with some of them executed by volunteers (Spec Core Team, Code Core Team) that are not paid by The Matrix.org Foundation as per its rules, it's not obvious to me how The Matrix.org Foundation envisions to spend those additional donations. And again, due to lack of reporting, the community does not even know how donations have been spent in the past.

Please make the extra effort of providing some transparency here, both for spendings of the last year(s) and the plan for spending of future donations. The official accounts raise more questions than they provide answer (for example: why is there more than 100k£ owing in from New Vector Limited?).

Thanks.

**Comment by ara4n** [comment]
I've tried to enumerate the functions of the Foundation in the set of bullet points at the beginning of https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022, and spell out the financial situation.  Currently the vast majority of the cost of this (~95%) is borne by Element (the trading name for New Vector Ltd, the company set up by the Matrix core team to try to fund Matrix development), where manpower is donated to the Foundation (and so doesn't show up on the books).  The way it works is that all IP that Element employees commit to https://github.com/matrix-org has its copyright assigned unilaterally to the Foundation in an asset lock.  You can see the amount of work that is, and estimate the donated costs, via something like Cauldron.io: https://cauldron.io/project/3729.

Historically we have had *very* little manpower on the financial side to do reporting beyond the legal minimum; we've been spending our $ on building Matrix rather than hiring non-profit finance people.  However, as part of operationalising the Foundation to gather and distribute larger donations (as per https://matrix.org/blog/2022/12/01/funding-matrix-via-the-matrix-org-foundation) we're addressing this: we are trying to hire both a Executive Director for the Foundation to be employed by the Foundation and be responsible for this independently from Element - and put all the legal & financial frameworks in place to handle the large amounts of $ involved.  Element is also donating non-profit finance manpower to plug the gap until an Executive Director is in place.

Hope this provides a bit more clarity; we'll update here as things progress further.

**Comment by alohapersona** [comment]
Hi @ara4n. Thanks a lot for your reply.

> I've tried to enumerate the functions of the Foundation in the set of bullet points at the beginning of https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022

Can you please outline how the bullet points described match the [Functions of the Foundation as of MSC1779](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/1779-open-governance.md#functions-of-the-foundation)? To me it seems a lot of the items listed are not really in scope of the work of the Matrix Foundation.

> Currently the vast majority of the cost of this (~95%) is borne by Element (the trading name for New Vector Ltd, the company set up by the Matrix core team to try to fund Matrix development), where manpower is donated to the Foundation (and so doesn't show up on the books).

I understand that Element is donating large amounts of non-monetary assets to the Foundation. For some of these donations, the transfer is supposed to happen as per MSC1779, for others this seems to not be the case. In any case, those donations are obviously very generous and highly appreciated.

The way I read your messages, it sounds as if the idea is to have the work currently done by Element and then donated to Matrix Foundation be done by the Matrix Foundation in the future (short term probably by paying Element), which seems to not be covered by MSC1779. It also significantly changes the position of Matrix Foundation from a standards organisation (like IETF or W3C) with some additional tasks (like hosting the matrix.org home server and ensuring the existence of a reference implementation). 

Would you please explain as to why this approach was chosen rather than
- having the Matrix Foundation continue to act as a mostly standards organisation
- put new developments under copyright of Element (instead of Matrix Foundation) which doesn't make a practical difference for users and developers as long as the license remains the Apache License.
- allow users to donate to Element to support the development of libraries, SDKs and the element client

I think such an approach would be much more compatible with MSC1779 and won't persistently raise questions about who to receive how much money from the Matrix Foundation for their developments (e.g. would the independent libQuotient developers be receiving the same funding as the element-hired developers of matrix-js-sdk?).

> Hope this provides a bit more clarity

Thanks, It does give some idea on the plans for the future of the Matrix Foundation.

However it doesn't really give any insights into previous years and current financials of the Matrix Foundation. Even very small documentation would already be very much appreciated. Could be as easy as:
|||
|:--|--:|
| Account balance at beginning of fiscal year: | XXX £ |
| Income from donations: | + XXX £ |
| Income from sponsorships: | + XXX £ |
| Other income: | + XXX £ |
| Expenses for hosting matrix.org: | - XXX £ |
| Expenses for promotion: | - XXX £ |
| Expenses for ensuring existence of reference implementations: | - XXX £ |
| Expenses for administration and organisation: | - XXX £ |
| Other expenses: | - XXX £ |
| Asset re-evaluation (for crypto): | +/- XXX £ |
| Account balance at end of fiscal year: | **XXX £** |

Most of this should probably be in easy reach as they'd likely be needed to create the accounting report legally required.

**Comment by MTRNord** [comment]
> put new developments under copyright of Element (instead of Matrix Foundation) which doesn't make a practical difference for users and developers as long as the license remains the Apache License.

While I agree with the rest of you comment I object with this part. As a developer this makes a major difference. Element is a for profit company and while I trust them that trust not only may change but also the motives of element could change at some point which endangers there reference implementations.

Having them legally in the non profit foundation is imho safer than having it in element even if element is the one doing a majority of work on it currently.

This mostly boils down to scenarios like elasticsearch where suddenly for profit reasons the license got changed. The foundation wouldn't have a reason to do that therefor it is somewhat more secure if the code is in their hands imho.

Obviously there is always a chance that something changes about this assumption. But I would be happy if it stays at the foundation since that makes the reference implementations separate from the for profit oriented companies.

**Comment by alohapersona** [comment]
> Having them legally in the non profit foundation is imho safer than having it in element even if element is the one doing a majority of work on it currently.

Let me explain my thinking here:
- As soon as Element publishes code under the terms of the Apache License, everyone can use it under these terms for eternity (Apache License is irrevocable, perpetual and sublicensable). Publication under the copyright of Matrix Foundation makes little difference (ignoring the name in the copyright notice).
- Element can, at any point, change the license of their newly produced code. If they published under Matrix Foundation, they first need to fork the code from Matrix Foundation. As Apache License is not a copyleft license, Element can fork code from Matrix Foundation with virtually no restrictions.

Or to phrase it differently: We're at the mercy of Element for them continuing to publish their code under Apache License. Whether that happens under the name of Matrix Foundation or Element doesn't really change anything about this.

**Comment by alohapersona** [comment]
Hello. It's me again.

The latest accounts statement as required by law, was published a few days ago.
- According to the statement, the book value of cryptocurrency assets did not change between Oct 31, 2021 and Oct 31, 2022. This is questionable, to say it mild, given that there have been incoming cryptocurrency transactions publicly visible on the respective published blockchain addresses of The Matrix.org Foundation.
- According to the statement, there now are £270k cash at bank or in hand. Does that mean that the issue of not having bank account has been resolved at the money from Patreon was paid out?
- The £116k owing from New Vector is now no longer in the book. Did New Vector pay £116k to The Matrix.org Foundation, and if so, for what reason?
- There now is a credit of £114k owed to New Vector within a year (that is, before Oct 31) in the book. What is behind this amount? Did New Vector bill any services to The Matrix.org Foundation and if so, can you provide more details on this?

Again, @ara4n, I think it would be very beneficial to the foundation if it was publishing any reports, even if as simple as what I suggested above. It will make people understand what donations are being used for and thereby likely will boost the donations the foundation receives.

**Comment by alohapersona** [comment]
Hello. It's me again.

- Turns out Oct 31 has passed by. As I wrote last time, The Matrix.org Foundation for some reason owed New Vector Ltd. £114k till that date. Did The Matrix.org Foundation pay that amount to New Vector Ltd, and if so, what was the service billed by New Vector to Matrix.org Foundation that entitled them to such payment?
- On Nov 2, another 0.72 BTC (~25k USD) and 35.2 ETH (~70k USD) were moved from the foundation's cryptocurrency wallets to Kraken. Again, what is the reason for this transfer, given that it was previously said that the foundation would be holding those for rainy days and (last we know) the foundation has more than £100k on the bank?
- With the move to Donorbox there is now even less transparency on the financials of The Matrix.org Foundation, as incoming donations are not visible to the public there (which was possible with cryptocurrencies, LiberayPay and Patreon before). Do you plan to provide public insights (preferably with live data) to the Donorbox donation amounts?

Not-so-fun note for @MTRNord: Seems like putting Synapse and Dendrite under Apache License 2.0 in name of The Matrix.org Foundation has not stopped or hindered New Vector Ltd. in any way to change license and ownership of those projects. Sadly proves me right that it is really not a big thing.

@ara4n and @joshsimmons (who I guess is now responsible for this): This thread has multiple serious unanswered questions and partially conflicting statements. The way The Matrix.org Foundation publicly handles its financial responsibilities as a mostly donation-funded organisation seem a huge risk to the whole Matrix protocol to me - even more with further things being moved to the for-profit New Vector Ltd.

Looking forward to your replies.

**Comment by quantumpacket** [comment]
> With the move to Donorbox there is now even less transparency on the financials of The Matrix.org Foundation, as incoming donations are not visible to the public there (which was possible with cryptocurrencies, LiberayPay and Patreon before).

At this point I would assume this was done with the intention of reduction in transparency.

**Comment by joshsimmons** [comment]
Hi @alohapersona, thanks for the tag and for being a persistent advocate for transparency. Transparency is something we should expect from any open source foundation, and I am committed to delivering on that as the Foundation's first Managing Director.

That said, it is the weekend :-) You'll hear more from me tomorrow.

**Comment by alohapersona** [comment]
Hi @joshsimmons. Thanks for the swift comment. Also be certain that I recognize that for any of this, you can't be blamed as you just took over responsibilities and probably didn't even know exactly what you are signing up for at the time.

**Comment by joshsimmons** [comment]
Thanks for your understanding, @alohapersona. As I'm now spending part of my weekend monitoring a pile on over on the other website, I'll take the opportunity to cross-post what I've said there:

I get that we're operating in a low trust environment, and you don't yet have any reason to trust me.

I will say this: I'm not here to carry water for past performance. I _am_ here to build the foundation into the organization the community deserves, and I have a track record when it comes to making organizations more responsive to the communities they serve.

Not only will I respond to the questions and concerns, I am actually working on systemic solutions that will provide transparency as a matter of course. Of course, these are just words for now. Judge me by my actions in the days and months ahead.

Know that I do aim to earn your trust as an individual and as an organization, _as well as_ to build the foundation into the kind of organization that, by merit of its rules, processes, and community representation in governance, doesn't require us to be trusting in the first place.

More to say tomorrow!

**Comment by joshsimmons** [comment]
Hi @alohapersona, I’ve included answers to your specific questions below, but first I wanted to share some context.

You can expect the Matrix.org Foundation to grow and evolve like other open source foundations, in terms of incrementally staffing up, increasing transparency and community governance, and establishing programs that support the ecosystem.

There are a lot of great examples we can learn from, though the manner in which we are starting adds some complexity. It’s a lot cleaner to start with a small balance and some IP than to inherit massive pre-existing commitments like running the matrix.org homeserver. The added complexity also makes things more difficult, and it’s fair to say the path to now has reflected that difficulty level.

The Foundation hit a key milestone this year when it delivered on its promise to make its first hire. If you look at the landscape and the history, you’ll find that the first hire makes a radical difference in delivering on the kinds of things I see you’ve been seeking for the last four years.

### Setting expectations

So, I’d like to set some expectations. The Foundation’s fiscal year, as you know by now, runs from November 1 to October 31, and it takes time to close the books. Especially when we’re still separating out our infrastructure and relying on people for whom the Foundation isn’t their first priority. After we close our books, two things happen: we send them off to the tax accountants, and I begin preparing our annual report. That annual report will cover revenue, expenses, and programmatic activity, all contextualized with our vision and where we are on the path to realizing our vision.

The timeline is a little fuzzy, but I’d expect that annual report to land around March or April.

And we should expect further maturation/normalization of process each successive year: when we seat our first elected Governing Board in 2024 Q2, I’ll finally have a board of community representatives to review and approve my budget! With that, we’ll start to see both a public budget before each financial year begins, and an annual report after it ends.

### Getting involved

Ultimately, the project is to build this Foundation up into an independent, transparent, self-sustaining organization with community-driven governance at every layer. We will need continued mutual accountability to stay the course on the way to achieving that vision. To wit, I welcome call-ins and incisive questions, and also stress the need to be charitable with each other. This work is hard, it is painfully incremental, and it happens over the scale of years and decades.

I do not, however, welcome escalation to Hacker News with bad faith editorializing, which brings out the worst in people and leaves me wasting time confronting FUD rather than tending to forward-looking work that truly serves the community. Starting fires like that isn’t helpful.

For those who are invested in Matrix and want greater visibility into the Foundation’s activities, I invite you to join the [Office of the Matrix.org Foundation](https://matrix.to/#/#foundation-office:matrix.org) room I launched last month. We have a lovely community of people there who provide input, offer accountability, and join together to celebrate even the small successes on our collective journey to organizational maturity and self-governance.

I hope this helps you and all those who read this to understand where we are, where we are going, why things are the way they are, and how to get involved.

With gratitude,
Josh Simmons
Managing Director of the Matrix.org Foundation

ad astra per aspera 🚀

### To your specific questions

#### Payments to Element/New Vector Ltd

The Foundation has a Master Services Agreement (MSA) with Element which includes administrative services, operation of the matrix.org homeserver, and secondment of several personnel whose responsibilities include advocacy, program management, standards work, and trust and safety. The MSA also includes provisions for the Foundation to reimburse Element for any bills they pay on our behalf.

What you are seeing is that the Foundation is beginning to realize the actual cost of its operations. As the Foundation grows and begins to stand on its own two feet, you can expect many of these line items will become internal costs rather than costs bundled into our MSA. Put plainly, we’re still in the process of becoming independent and we aim to eventually staff many of these functions ourselves – whether through hiring employees, or contracting individuals or other companies.

#### Cryptocurrency

At the risk of upsetting folks who like cryptocurrency, I deemed it irresponsible for the Foundation to be sitting on highly volatile speculative assets – the value could go to zero, or the exchanges could go bust. Indeed, the value has already plummeted from a high of 800K GBP. That’s not a sound approach to a rainy day fund, because, say what you will about the current banking system, our deposits are at least guaranteed there. To that end, I’ve begun liquidating our cryptocurrency and have unpublished our wallet IDs so we are less likely to receive donations this way.

To your question back in August about the book value of the cryptocurrency being reported as unchanged between 2021 and 2022: that must’ve been an error. 

#### Donorbox, Patreon, and Liberapay

Yes, we plan to provide information about donations in our reports, including aggregate figures and some analysis, much the way we see other open source foundations doing so. No, we will not provide live data.

We’re consolidating on Donorbox because it has lower transaction fees. Many early stage nonprofits make the mistake of maintaining too many different channels for donations, which introduces more overhead than it’s worth – not just in accounting, but in monitoring all the inboxes, servicing each of the platforms, and trying to maintain an authoritative record of who the donors are.

**Comment by alohapersona** [comment]
Hi @joshsimmons. Thanks a ton for your comprehensive reply.

> The timeline is a little fuzzy, but I’d expect that annual report to land around March or April.

I think it's totally fine and reasonable to have annual reports land 6 months after end of fiscal year.

Do I understand it correctly, that you do not plan to provide any kind of report for previous years, even though seemingly there have been significant money flows in and out of the foundation in those years?

> escalation to Hacker News

Under the previous regime of the foundation, the only way to get a response to this thread was to escalate it to more public venues. I should have had given you a chance to prove you handle this differently before escalating, but I failed to realize that things are already handled by you.

> The Foundation has a Master Services Agreement (MSA) with Element

Can you share how much was paid to New Vector under this agreement, if possible on a per-year basis and categorized (e.g. in categories like "maintaining matrix.org homeserver", "promotion", "maintaining the standard/specification", "administration/legal")? Given that New Vector is a for-profit entity with a special role in the Matrix ecosystem and one of the purposes (if not the main purpose) of the foundation is to decouple Matrix from New Vector, I strongly believe the relationship with New Vector is where many people would want to see more transparency.

>  I deemed it irresponsible for the Foundation to be sitting on highly volatile speculative assets

This decisions (even if a little controversial) indeed gives the feeling that the foundation is now actually being managed responsibly, so thanks for sharing that note.

> that must’ve been an error

Will you follow up on this with accountants or whoever is responsible for that? I don't know UK company law, but I bet erroneous account statements must be corrected ASAP.

> No, we will not provide live data.

Would you at least consider to provide updated snapshots of donation information more regularly than in the annual report (e.g. monthly). I bet this can even be automated, so that no manual work is needed once set up.

> evolve like other open source foundations

A little off-topic to this thread, but this sounds as if you see The Matrix.org Foundation more as an open source organization and less as a standards organization. This seems to not fully match what is outlined in MSC1779. How do you see the Matrix.org Foundation's position to non-free clients and servers (e.g. I see that [Beeper](https://www.beeper.com/) is not listed on the [clients page on matrix.org](https://matrix.org/ecosystem/clients/))? Can developers of proprietary matrix components participate in the standardization process? Could they get a seat in the Spec Core Team or be a Guardian?

Again, thanks for making an effort to handle my messages. I do know this stuff can be tedious and I also bet when taking over the leadership of the foundation, it wasn't really in best shape, so I appreciate any attempt to improve things from the status quo.

**Comment by joshsimmons** [comment]
Thanks for the follow up questions, @alohapersona, and I deeply appreciate the spirit in which you received my call-in about escalating. I'm on holiday for the rest of the week for $ComplicatedUSHoliday but will follow up with you on this thread early next week!

**Comment by joshsimmons** [comment]
Thanks for your patience! Hope you had a great week/end, @alohapersona. Responses in line:

> Do I understand it correctly, that you do not plan to provide any kind of report for previous years, even though seemingly there have been significant money flows in and out of the foundation in those years?

It’s just not a priority at this time, and getting around to it would require us to be in substantially better condition. We are in extreme triage mode and housekeeping for past years doesn’t make the cut.

> Can you share how much was paid to New Vector under this agreement [...] ?

Absolutely, I will gladly share all of that information with appropriate context in our 2023 Annual Report. For now I can say that mostly things have been donated and much still is, our first significant invoice from Element was in 2022, and only in 2023 are we beginning to shoulder the full costs of our bills and staff time. To give you a sense of magnitude, we’re talking about several full-time staffers and numerous part-timers covering the aforementioned functions.

> Will you follow up on this with accountants or whoever is responsible for that? I don't know UK company law, but I bet erroneous account statements must be corrected ASAP.

I appreciate your concern and can assure you that everything that needs to be done will be done :-) It is my job, and the job of the accounting firm we work with, to ensure that everything is above board and in compliance with applicable laws and regulations.

> Would you at least consider to provide updated snapshots of donation information more regularly than in the annual report (e.g. monthly). I bet this can even be automated, so that no manual work is needed once set up.

None of the open source foundations I’m familiar with do this kind of thing. But sure, if there was a compelling reason and the resources to do it, of course I’d consider it.

> [T]his sounds as if you see The Matrix.org Foundation more as an open source organization and less as a standards organization.

I see it as necessarily both – as a standards organization for the Matrix protocol, and as an open source foundation for the ecosystem around it. I think we see this duality in many places, for instance the Python Software Foundation and Rust Foundations both support the wider ecosystem as well as look after the process by which their respective languages evolve through PEPs or RFCs.

> How do you see the Matrix.org Foundation's position to non-free clients and servers (e.g. I see that Beeper is not listed on the clients page on matrix.org)? 

Oh, I love this question. And, I should say, I also generally appreciate your questions: they provoke important conversations that we need to keep having as a community to chart our path.

My view is that we are obligated to focus on and prioritize FOSS clients, servers, etc, based on our nonprofit status as well as on our mission and the Matrix manifesto. That said, I don’t think that precludes recognizing proprietary implementations. Indeed, while five out of six of our Ecosystem pages exclusively list FOSS projects, one of them – Hosting – lists organizations that offer paid solutions and may use proprietary software somewhere in the mix. I reckon it’d make sense to list Beeper somewhere in our Ecosystem pages, I’m just not sure where yet.

> Can developers of proprietary matrix components participate in the standardization process? Could they get a seat in the Spec Core Team or be a Guardian?

Yes and yes, though the second “yes” is more theoretical because we don’t yet have community-driven processes by which such a person would be seated. Critically, though, they would need to abide by the rules and mission of the Foundation which bind us to FOSS and.

Further, when it comes to evolving the standard, I believe the Spec Core Team requires an open source implementation before an MSC can be approved and merged in – such that it can be tested, reviewed, and serve as an example for those who are developing FOSS projects that speak Matrix. That would remain true even for those who are developing proprietary components.

**Comment by turt2live** [comment]
👋 jumping in on the last point there in two parts:

> > Can developers of proprietary matrix components participate in the standardization process? Could they get a seat in the Spec Core Team or be a Guardian?
> 
> Yes and yes, though the second “yes” is more theoretical because we don’t yet have community-driven processes by which such a person would be seated. Critically, though, they would need to abide by the rules and mission of the Foundation which bind us to FOSS and.

To clarify, MSCs in particular *must* be submitted in line with the [Guiding Principles](https://spec.matrix.org/proposals/#guiding-principles), where fully-proprietary developers may find their ethos challenged.

> Further, when it comes to evolving the standard, I believe the Spec Core Team requires an open source implementation before an MSC can be approved and merged in – such that it can be tested, reviewed, and serve as an example for those who are developing FOSS projects that speak Matrix. That would remain true even for those who are developing proprietary components.

"Implementation" of an MSC is difficult to describe. Often times it's actual real code, but sometimes it can be theoretical. We do not have a requirement that the code (or theory) be open source, but rather evidence that the feature is works well at a pragmatic scale. Each MSC has its implementation requirements evaluated differently, though the common trend is *at least* one client and one server implementation, either as a PR, commit, or repo (if large enough to be its own project). 

These implementations *can* be proprietary if there's enough evidence externally to show that the feature is working as designed in the MSC and has significant adoption, though the SCT would be hesitant to accept that implementation as the sole satisfying requirement - further, open source, implementations would be required. That open source evidence can be as simple as a passing [Complement](https://github.com/matrix-org/complement) test against the proprietary system, for example, as the test's existence demonstrates that other compliant implementations can be created.

For the non-code implementations, consider [MSC4077](https://github.com/matrix-org/matrix-spec-proposals/pull/4077) - it changes the process *and* the spec, but does not have an implementation. It also doesn't have the capability to have a code-based implementation either. It's "implementation" is the theory that someone else understands best practice more than we do, so we're adopting their process into ours. If we required a code implementation for the MSC, it'd be largely pointless as all it'd demonstrate is that a client can use [`<s>` instead of `<strike>`](https://github.com/matrix-org/matrix-spec/issues/1492). This is proven implicitly when we consider a Find & Replace All function in an IDE could "implement" such a proposal. Or in other words, our "pragmatic scale" for such a change is assuming a client developer can write a trivial commit.

**Comment by joshsimmons** [comment]
Hi folks! Keeping this issue alive and open at least until we get that Annual Report out in April. Wanted to jump in with an update, as today we published some high level financial figures as we launch a new fundraiser and begin documenting the full scope of the Foundation's work as well as our emerging roadmap.

I recommend [reading the post](https://matrix.org/blog/2024/01/2024-roadmap-and-fundraiser/), but wanted to share some rough figures here:

* 2022 revenues were £82K
* 2023 revenues were £364K
* 2023 expenses were £440K

Last year's expenses were not representative of the full cost of running the Foundation though, and we expect annual expenses to stabilize just north of £900K. This comes down to us increasingly paying our own bills, instead of Element paying them for us – though Element still donates £3M+ in core development annually.

Breaking down that £900K figure, our 2024 budget looks something like:

* £550K in staffing 4 full-time people covering management, advocacy, standards development, and trust and safety + many external part-time staff for core project and bridge maintenance and infra ops
* £250K in servers and other third-party providers
* £100K in administrative costs covering legal, accounting, and compliance

Our plan for closing the budget gap is to focus on bringing in more recurring revenue through our membership program, and temporarily shore things up where possible with grants.

If we don't make good headway by ~Q3, we'll start implementing contingency plans to reduce our scope of work to make it sustainable. Fundamentally, if nothing else, the Foundation will fall back to stewarding the spec ... but we'd like to continue investing in advocacy and standards work as well as the operation of public servers, bridges, and bots.

**Comment by magcius** [comment]
Has the annual report been published?

**Comment by MTRNord** [comment]
Any update? :/

**Comment by turt2live** [comment]
Update from the Office of the Foundation room: https://matrix.to/#/!sWpnrYUMmaBrlqfRdn:matrix.org/$LiX-5SIydfgPnBr-ryeoP5bbBRn1s_nr9pN7KrIEb_w?via=matrix.org&via=element.io&via=envs.net

Hopefully the report should be ready soon, though may be missing granular detail (sorry).

**Comment by l29ah** [comment]
https://matrix.org/blog/2025/02/crossroads/ some info got there

**Comment by bluesomewhere** [comment]
@l29ah thank you for linking that here! And I'm happy to share more details.

Process wise: we finally have a Governing Board and a Finance & Fundraising Committee, and I shared rough figures with them last night. I'm gunna work through things with the Committee when we meet in early March, and then share more. I'm really sorry it's taken so long.

And in terms of numbers, out of the $1.2M annual costs we mention there, here's the rough breakdown:

* $360K/yr – Trust & Safety
* $250K/yr – Other folks, covering program management, legal, compliance, finance, IT, security, dev
* $240K/yr – Server and SRE
* $170K/yr – My salary, covering fundraising, open governance, community relations, biz ops, and people mgmt
* $150K/yr – Matrix Conf + other events
* $30K/yr – Assorted expenses like travel, lodging, productivity infra

**Comment by HarHarLinks** [comment]
The scope of the Finance & Fundraising Committee is defined through [its charter](https://matrix.org/foundation/governing-board/committees/#finance-fundraising) and does not include generating the report.

**Comment by mpeter50** [comment]
Maybe my expectations are too high, in which case please let me know, but the level of detail in the 2025 annual report was underwhelming, so I am hoping that improves this time around.

Also, I am asking with genuine interest and no experience in management, what takes that much time in making the report? Is it that hard to collect all the documents about the spendings?

**Comment by alohapersona** [comment]
I think this fully resolves the issue.

I do have a question though and not sure what would be the best place to ask for it: The way Element's contributions are brought up in the report are questionable at least. I happen to have been volunteer auditor for a non-profit, and the way I know how in-kind contributions are reported, is by estimating the fair market value of them and record them both as revenue and expense. It basically was the same as if the in-kind contribution value was donated in cash and then that cash used to buy what was actually the in-kind contribution.

You may ask why this is relevant. I think it's crucial to have a better understanding of the financial situation of the foundation, because those in-kind contributions would potentially need to be paid through other donations if Element was no longer available to provide them.

In other words, if in-kind contributions from Element are ~500k GBP, then costs of Matrix Foundation is more in the realm of 1.4M GBP insead of the 900k GBP shown in the report. Revenue would consequently need to be recorded as 1.1M GBP instead of the 600k GBP today, with percentages adjusted (Platinum member: 45%, Gold member: 28%, Silver members: 8%, Donations: 10%, Event sponsors: 7%, Ticket: 2%). For the cost side, it's much harder to figure out the adjusted percentages, largely because I don't know exactly where in-kind contributions fall in. This also shows that having those in-kind contributions better reflected helps to better understand the financials and to learn where there is the highest potential for cost reductions.
